NAMESPACE

```diff
@@ -470,6 +471,7 @@ export(vec_math)
 export(vec_math_base)
 export(vec_na)
 export(vec_na_along)
+export(vec_names)
```

Comment:
So this is the heir to `rlang::names2()`, right?

Reply:
Correct, though `names2()` is lower level and works with base R definition of vector (e.g. `colnames()` of data frames).

Reply:
`names2()` still has it's place - it has the invariant that `length(names2(x))` equals `length(x)`, where `vec_size(vec_names(x))` equals `vec_size(x)` (which is very different for data frames)

Reply:
💡 is it called `names2()` just because it's a second take on `names()`? Probably.

But I just realized you can say that, for a data frame, vec_names() gives names along dimension 1 and `names2()` gives names along dimension 2.

Reply:
Though that doesn't work with matrices and simple vectors don't really have 2 dimensions (and if taken as a column vector, `names2()` returns the names of dim 1 rather than 2).